calculator.test.js

```diff
@@ -6,16 +6,26 @@ var calculator = require('./calculator');
 
 describe('Calculator', function() {
   
-  it('should add two numbers', function() {
-    assert.equal(calculator.add(1,1), 2);
+  it('should add numbers', function() {
+    assert.equal(calculator.add(2,2,2,2,2), 10); // test will fail
```

Comment:
HI vu, what i mean on this is that the method was not yet updated to handle the multiple arguments. 

    On Friday, June 24, 2016 12:27 PM, Vu Nguyen <notifications@github.com> wrote:
 

 In calculator.test.js:> @@ -6,16 +6,26 @@ var calculator = require('./calculator');
>  
>  describe('Calculator', function() {
>    
> -  it('should add two numbers', function() {
> -    assert.equal(calculator.add(1,1), 2);
> +  it('should add numbers', function() {
> +    assert.equal(calculator.add(2,2,2,2,2), 10); // test will fail
Accordingly to comment, test will fail ?, I think it should pass to verify implement correctly ?—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.